cmd/pod/getPod.go

```diff
@@ -38,14 +52,15 @@ var GetPodCmd = &cobra.Command{
 					fmt.Sprintf("%d", p.ContainerDiskInGb),
 					fmt.Sprintf("%d", p.VolumeInGb),
 					fmt.Sprintf("%.3f", p.CostPerHr),
+					fmt.Sprintf("%s", strings.Join(ports[:], "\n") ),
```

Comment:
are these supposed to be separated by newlines?

Reply:
no